qiskit/ignis/verification/randomized_benchmarking/experiments/analysis.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+from scipy.optimize import curve_fit
+from typing import Dict, Optional, List, Union
+from qiskit.result import Result, Counts
+from qiskit.providers import BaseJob
+from qiskit.ignis.verification.tomography import marginal_counts
```

Comment:
Perhaps it's not good that RB refers to tomography, maybe put `marginal_counts` in a joint utils directory or move to Terra?